klippy/extras/palette2.py

```diff
@@ -638,10 +638,12 @@ def _run_Smart_Load(self, eventtime):
 
     def get_status(self, eventtime=None):
         status = {
-            "ping": self.omega_pings[-1],
+            "ping": None,
             "remaining_load_length": self.remaining_load_length,
             "is_splicing": self.is_splicing
         }
+        if self.omega_pings:
+            status.update({ "ping": self.omega_pings[-1] })
```

Comment:
Why use that form as opposed to the more common `status["ping"] = self.omega_pings[-1]`?

-Kevin

Reply:
Yeah that would make a lot of sense. I blame a lack of coffee when I wrote this.

Reply:
FYI, trivial change so I rebased.